signalmlb.lua

```diff
@@ -0,0 +1,40 @@
+-- A simple Lua module for HAProxy that sends signals to the marathon-lb process
+
+function run(cmd)
+  local file = io.popen(cmd)
+  local output = file:read('*a')
+  local success, _, code = file:close()
+  return output, success, code
+end
+
+function send_response(applet, code, response)
+  applet:set_status(code)
+  applet:add_header("content-length", string.len(response))
+  applet:add_header("content-type", "text/plain")
+  applet:start_response()
+  applet:send(response)
+end
+
+core.register_service("signalmlbhup", "http", function(applet)
+  local _, success, code = run("pkill -HUP -f '^python.*marathon_lb.py'")
```

Comment:
One concern here is what happens if they're running multiple MLBs within the same PID namespace.